index.html

```diff
@@ -182,76 +237,151 @@ <h1 class="work-sample"> Profesional Art Printing Data </h1>
 			<a class="see" href="#"> See Project</a>
 		</div>
 	</div>
+
+	<!-- destop cards -->
+	<div class="card hide">
+		<div class="row first-row">
+			<div class="cards">
+				<img src="assets/images/first-col.png" alt="rec">
+				<a class="see" href="#"> See Project</a>
+			</div>
+			<div class="cards card-2">
+				<h1 class="work-sample"> Data Dashboard Health care </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+			<div class="cards card-3">
+				<br><br>
+				<h1 class="work-sample"> Website Protfolio </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+		</div>
+		<div class="row first-row">
+			<div class="cards card-1">
+				<h1 class="work-sample"> Profesional Art Printing Data </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+			<div class="cards card-2">
+				<h1 class="work-sample"> Data Dashboard Health care </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+			<div class="cards card-3">
+				<br><br>
+				<h1 class="work-sample"> Website Protfolio </h1>
+				<p class="work-sample-text">
+					A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard
+				</p>
+				<ul class="tags">
+					<li class="tag">html</li>
+					<li class="tag">bootstrap</li>
+					<li class="tag">ruby</li>
+				</ul>
+			</div>
+		</div>
+	</div>
+	</div>
```

Comment:
- [ ] I cant see any see a project on these projects. Please add it

![image](https://user-images.githubusercontent.com/38635158/173109684-62bee9ec-796b-4cc3-aa9e-da111da6bcf4.png)
___
Original
![image](https://user-images.githubusercontent.com/38635158/173109822-e3fd0758-eaf0-4553-908a-c46b5f8e6e6b.png)

Reply:
Thank you for the corrections. Please check it by Inspect -> mobile devices -> Responsive